selfdrive/car/isotp_parallel_query.py

```diff
@@ -7,9 +7,13 @@
 from selfdrive.boardd.boardd import can_list_to_can_capnp
 from panda.python.uds import CanClient, IsoTpMessage, FUNCTIONAL_ADDRS, get_rx_addr_for_tx_addr
 
+class RX_OFFSET:
```

Comment:
just make a variable called `DEFAULT_RX_OFFSET` in fw_versions.py and hardcode the VW offset, don't want to have make-specific stuff here

Reply:
Done